conda/environments/all_cuda-125_arch-x86_64.yaml

```diff
@@ -140,5 +139,6 @@ dependencies:
   - nemollm==0.3.5
   - pymilvus==2.3.6
   - pytest-kafka==0.6.0
+  - sentence-transformers==2.7
```

Comment:
Add a comment why we are installing this via pip and not conda so we know if we can move back to conda in the future